packages/teleport/src/Welcome/Welcome.tsx

```diff
@@ -21,13 +21,21 @@ import history from 'teleport/services/history';
 import LogoHero from 'teleport/components/LogoHero';
 import cfg from 'teleport/config';
 
+import { CaptureEvent, userEventService } from 'teleport/services/userEvent';
+import useToken from 'teleport/Welcome/useToken';
+
 import { NewCredentials } from './NewCredentials';
 import { CardWelcome } from './CardWelcome';
 
 export default function Welcome() {
   const { tokenId } = useParams<{ tokenId: string }>();
+  const tokenProps = useToken(tokenId);
```

Comment:
sorry I should've explained the reason behind this whole welcome screen. if my understanding is correct, we had problems with crawlers clicking on the invite/reset link before the user did (email link), which would read the token and generate the QR code. By the time user got to it, the QR code will expire. This welcome screen prevented the crawler from reading the token (and generation of the QR code). This was an issue for users who opted for second factor `otp`. @rudream can you double check please?

history: https://github.com/gravitational/webapps/pull/479

Reply:
Oh yes great context

Reply:
i think it might be a good idea to leave a comment on this component explaining why it exists (with a link to PR perhaps)

Reply:
I will remove this event for the first iteration and rethink how to solve this once we go live next week with the first round of events. I'll make a note to add a comment to the component at that time! Thank you for the history!

Reply:
That's correct @kimlisa, there was an issue where users who received invite links sent by email (ie for Cloud) would get `totp code invalid` errors when trying to sign up. The problem was caused by mail web crawlers who would open the link and trigger the TOTP QR code to be generated before the user did. Moving it to a second page and only allowing the web crawler to see a welcome screen solved the issue.